js/luno.js

```diff
@@ -112,11 +112,11 @@ module.exports = class luno extends Exchange {
             let reserved = parseFloat (balance['reserved']);
             let unconfirmed = parseFloat (balance['unconfirmed']);
             let account = {
-                'free': parseFloat (balance['balance']),
+                'free': 0.0,
                 'used': this.sum (reserved, unconfirmed),
-                'total': 0.0,
+                'total': parseFloat (balance['balance']),
             };
-            account['total'] = this.sum (account['free'], account['used']);
+            account['free'] = account['free'] - account['used'];
```

Comment:
This line looks like a typo, shouldn't it be 
```JavaScript
account['free'] = account['total'] - account['used']
``` 
?

Reply:
Yes. Sorry I will fix it now.

Reply:
Thanks!